docs/app/docs/devbox_examples/databases/postgres.md

```diff
@@ -9,13 +9,18 @@ PostgreSQL can be automatically configured by Devbox via the built-in Postgres P
 
 ## Adding Postgres to your Shell
 
-`devbox add postgresql glibcLocales`, or in your `devbox.json`, add
+You can install the PostgreSQL server and client by running`devbox add postgresql`. In some Linux distributions, you may also need to add `glibcLocales`, which can be added using `devbox add glibcLocales -p x86_64-linux,aarch64-linux`.
```

Comment:
nit: slight preference to use `--platform` instead of shorthand `-p` since this is documentation. Makes it more self-explanatory.